src/svpcvp.cpp

```diff
@@ -157,7 +157,9 @@ static int shortestVectorEx(IntMatrix& b, IntVect& solCoord,
     FPLLL_CHECK(result, "shortestVector: cannot compute an initial bound");
     maxDist.add(maxDist, ftmp1, GMP_RNDU);
   }
-
+  
+  if(flags & SVP_OVERRIDE_BND) maxDist.set_z(argIntMaxDist, GMP_RNDU);
```

Comment:
I think there should be an if/else block as it's easier to understand it as an alternative. Currently, this simply overwrites a computation done earlier and it's not clear to the reader if this is a potential bug or by design.